symplyphysics/definitions/wave_equation_in_one_dimension.py

```diff
@@ -38,48 +40,37 @@
     :code:`u(x, t)`
 """
 
-position = Symbol("position", units.length, real=True)
+position = symbols.position
 """
 Position, or spatial variable.
-
-Symbol:
-    :code:`x`
 """
 
-time = Symbol("time", units.time, positive=True)
+time = clone_as_symbol(symbols.time, positive=True)
 """
 Time.
-
-Symbol:
-    :code:`t`
 """
 
-phase_speed = Symbol("phase_speed", units.velocity, real=True)
+phase_speed = clone_as_symbol(symbols.phase_speed, positive=True)
 """
 :doc:`Phase speed <laws.waves.phase_velocity_from_angular_frequency_and_wavenumber>` of the wave.
-
-Symbol:
-    :code:`v`
 """
 
 definition = Eq(
     Derivative(displacement(position, time), position, 2),
     Derivative(displacement(position, time), time, 2) / phase_speed**2,
 )
-r"""
-:code:`Derivative(u(x, t), (x, 2)) = (1/v^2) * Derivative(u(x, t), (t, 2))`
+"""
+:laws:symbol::
```

Comment:
`displacement` is not `SymbolNew`. Autogeneration may fail to properly process it. Use manual description.

Reply:
You're right, I forgot